saleor/graphql/order/mutations/order_grant_refund_create.py

```diff
@@ -182,6 +187,32 @@ def validate_input(cls, input: dict[str, Any]):
                 }
             )
 
+    @classmethod
+    def _resolve_refund_reason_instance(
+        cls, /, reason_reference_id: str, refund_reason_reference_type: str
+    ):
+        reason_reference_pk = cls.get_global_id_or_error(
+            reason_reference_id, only_type="Page", field="reason_reference"
+        )
+
+        try:
+            reason_reference_instance = Page.objects.get(
+                pk=reason_reference_pk,
+                page_type=refund_reason_reference_type,  # type: ignore[misc]
```

Comment:
Non blocking: Do we really need this `type: ignore`. Looks like we can adjust typing/code to make mypy happy.